scoremtb/score/get_spellvocab_score.py

```diff
@@ -0,0 +1,68 @@
+# MTB Scoring
+
+# import the required packages
+import pandas as pd
+from scoremtb.util import util as ut
+
+def get_score_spell(task_data, dfid, type):
+    """
+    -----------------------------------------------------------------------------------------
+    
+    This is the function to get score 
+    
+    Args: 
+        task_data: taskdata downloaded from Synapse
+        dfid: metadata downloaded from Synapse
+        type: Task/Score, in this function, there are only three choices: Spelling, Vocab and PSM
+
+    Return:
+        Scoring dataframe
+    
+    -----------------------------------------------------------------------------------------
+    """
+    config = ut.get_config()
+    common_cols = ['healthcode', 'recordid', 'assessmentid', 'deviceInfo', 'startDate', 'endDate',
+                   'participantversion', 'clientinfo', 'sessionguid', 'taskStatus_val']
+    
+    if type == config['spell_type']:
```

Comment:
Couldn't this code be much simpler?

Have a dict:assesment2colums = {t_type : [theta, finalTheta_Double, SE]

theta_col, theta_double_col, se_col = assesment2colums[t_type]